src/components/SidePanelControlBar.tsx

```diff
@@ -103,20 +103,15 @@ const SearchBarComponent: FC<SidePanelControlBarProps> = ({
           )}
           <ThemeButton
             color="tertiary"
-            aria-label="Filter"
+            aria-label={filterCount === 0 ? "Filter Button" : `Filter Button ${filterCount} filters active`}
+            aria-roledescription=' '
```

Comment:
Keeping the semantic meaning of the button is more important than matching the suggested text order, so you can remove this line ``aria-roledescription=' '`. Also `aria-roledescription` should never be an empty string, it should be a semantic role that you apply to divs or other HTML elements that have no existing role.

For the aria-label you usually don't want to include words like "button" or "image." Ideally that will come from the component being coded semantically (a real button element instead of a div for ex).  So once you remove that aria-roledescription line, you can also remove "button" from both of the labels.

Reply:
Here is what it sounds like on VoiceOver/Chrome 
no filters selected
<img width="639" alt="Screenshot 2024-06-09 at 6 22 19 PM" src="https://github.com/CodeForPhilly/clean-and-green-philly/assets/7329799/2c01572f-0aa0-491d-acb0-50e82ac5678d">

2 filters selected
<img width="524" alt="Screenshot 2024-06-09 at 6 22 33 PM" src="https://github.com/CodeForPhilly/clean-and-green-philly/assets/7329799/a186c814-ce7a-4342-9d16-b8ea705714cc">